content/flowcontrol.article

```diff
@@ -92,7 +92,7 @@ https://golang.org
 (або змінюється тільки на дуже малу величину).
 Перевірте, чи це більше або менше 10 ітерацій.
 Спробуйте інші початкові вгадування для `z`, такі як `x` або `x/2`.
-Яка точність ваших результатів функції порівняно з [[/pkg/math/#Sqrt][math.Sqrt]] в стандартній бібліотеці?
+Яка точність ваших результатів функції порівняно з [[https://go.dev/pkg/math/#Sqrt][math.Sqrt]] в стандартній бібліотеці?
```

Comment:
I see 500 there. 

It looks like all go.dev/pkg/... requests are redirected to pkg.go.dev/...
Shall we replace them manually?

Reply:
What do you mean by `500`?
In https://go.dev/tour/flowcontrol/8 the link look exactly the same https://go.dev/pkg/math/#Sqrt and then they take care of the redirect. I think it's good to be consistent with the original.

Reply:
Good point!
Let's keep it the same way the original repo does.

P.S.: I don't know why but redirection in my Firefox was broken and I saw 500 errors each time I tried the https://go.dev/pkg/math/#Sqrt. However, that was a local issue and in Safari everything works well.